LTC_extracts_DRAFT.R

```diff
@@ -0,0 +1,137 @@
+
+#####################################################
+# Draft LTC Extract Processing Code
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description - Takes IT extract of LTC data.
+#              Renames and checks types of data.
+#              Flags variables which fall outside FY.
+#              Saves outputted flagged dataset.
+#####################################################
+
+## packages ##
+
+library(dplyr)
+library(stringr)
+
+
+## financial year in question ##
+FY <- 1718
+year <- convert_fyyear_to_year(FY)
+
+
+## Read data ##
+ltc_file <- readr::read_csv(
+  file = get_it_ltc_path(),
+  col_type = vroom::cols(
+    `PATIENT_UPI [C]` = col_character(),
+    `PATIENT_POSTCODE [C]` = col_character(),
+    `ARTHRITIS_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `ASTHMA_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `ATRIAL_FIB_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `CANCER_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `CEREBROVASC_DIS_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `CHRON_LIVER_DIS_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `COPD_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `DEMENTIA_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `EPILEPSY_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `HEART_DISEASE_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `MULT_SCLEROSIS_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `PARKINSONS_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `RENAL_FAILURE_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `CONGENITAL_PROB_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `BLOOD_AND_BFO_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `OTH_DIS_END_MET_DIAG_DATE` = col_date(format = "%Y/%m/%d %T"),
+    `OTH_DIS_DIG_SYS_DIAG_DATE` = col_date(format = "%Y/%m/%d %T")
+  )
+)
```

Comment:
Sorry the col date format was working for my acute file but this is different in the LTCs - it uses %d-%m-%Y instead. I'll commit the change